xmltodict.py

```diff
@@ -70,13 +70,16 @@ def __init__(self,
         self.force_list = force_list
 
     def _build_name(self, full_name):
-        if not self.namespaces:
+        if self.namespaces is None:
```

Comment:
Although a `defaultdict` essentially has infinite items, until an item is actually inserted (either explicitly or on first access), it has zero length and evaluates to `False`.